src/sentry/api/endpoints/organization_details.py

```diff
@@ -116,6 +124,38 @@ def update_organization_scenario(runner):
         )
 
 
+class TrustedRelaySerializer(serializers.Serializer):
+    def to_representation(self, instance):
+        return convert_dict_key_case(instance, snake_to_camel_case)
+
+    def to_internal_value(self, data):
+
+        try:
+            key_name = data.get(u"name")
+            public_key = data.get(u"publicKey", "")
+            description = data.get(u"description")
+        except AttributeError:
+            raise serializers.ValidationError("invalid-message")
```

Comment:
Please consider to fetch the `public_key` like this to also support an explicit `None` value. Otherwise, this would create an error below:

```py
public_key = data.get(u"publicKey") or ""
```

Reply:
yep

Reply:
done